packages/browser/src/index.test.ts

```diff
@@ -124,6 +124,19 @@ describe('LogtoClient', () => {
         'foo',
       ]);
     });
+
```

Comment:
Would be nice if we can add a test case here, to verify that using prompt login can actually get an access token

Reply:
Since UTs do not call actual Logto Core API, we cannot know whether the prompt value affects the response here.